contents/docs/plugins/build.md

```diff
@@ -124,51 +189,60 @@ async function processEvent(event, { config, cache }) {
 
 As you can see, the function receives the event before it is ingested by PostHog, adds properties to it (or modifies them), and returns the enriched event, which will then be ingested by PostHog (after all plugins run).
 
-### Global Parameters
+### processEventBatch function 
 
-PostHog automatically injects useful parameters in the `setupPlugin` and `processEvent` functions. These are:
+`processEventBatch` works just like `processEvent`, except it takes a batch of events at once, rather than one event at a time. It also returns an array of events. This is especially useful for plugins that export data out of PostHog, so that they do not need to make an HTTP request with every incoming event. 
 
-#### config
+> **Note:** Your plugin can use `processEvent` or `processEventBatch`. Currently, if both are present, `processEventBatch` will not run. 
 
-Gives you access to the PostHog config as described in `plugin.json` and configured via the PostHog interface.
+Here's the same example from above, except now using `processEventBatch`: 
 
-#### cache
-
-A way to store values that persist across `processEvent` calls. The values are stored in [Redis](https://redis.io/), an in-memory store.
+```js
+async function processEventBatch(events, { config, cache }) {
+    const counter = await cache.get('counter', 0)
+    cache.set('counter', counter + 1)
```

Comment:
There's a problem with this counter example. Since we can run many workers in parallel, it can happen that multiple tasks get `counter` as "1" and then both set it to "2". Async fun.

It's probably best to use `const counter = await cache.incr('counter')` that always makes sure the counter is incremented.

The code here demonstrates interesting things, but I wouldn't want people copying it.

Cache get/set might then be better shown in another example about caching some data, like currency rates.